### DISCUSSION TITLE
백엔드 api 연동시 네트워크 오류

### DISCUSSION THREAD

**Discussion by GBAJS754** [bucket-back/bucket-back-frontend#80] [General]
![image](https://github.com/bucket-back/bucket-back-frontend/assets/104294861/dc50b0bd-a6c1-4fdd-baa9-cdbe74a6a772)

**Comment by perfumeplaylist** [comment]
header에 잘못된 정보를 넣으면 403에러가 나온다!

**Reply by GBAJS754** [reply]
해당 문제를 해결한 PR입니다.
 https://github.com/bucket-back/bucket-back-frontend/pull/81
